contents/docs/web-analytics/web-vs-product-analytics.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: What is the difference between web analytics and product analytics?
+sidebar: Docs
+showTitle: true
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+[Web analytics](/web-analytics) is a streamlined experience for tracking and monitoring your most important website metrics. [Product analytics](/product-analytics) is a more powerful and flexible experience for tracking your product.
+
+When people talk about web analytics, they're generally referring to tracking visitors on your marketing website (e.g. `posthog.com` sessions). They care about things like page views, bounce rate, and top sources of traffic.
+
+Product analytics is the place to find insights about your application based on usage (e.g. `app.posthog.com` user behavior).
+
+<ComparisonTable column1="Web analytics" column2="Product analytics">
```

Comment:
Love the table

Reply:
Agreed. I can't take credit, though. I pulled the table from the FAQ.